c2corg_ui/templates/image/view.html

```diff
@@ -33,21 +35,34 @@
 </%block>
 
 <%block name="moduleConstantsValues">
-  module.value('mapFeatureCollection', {
-    "type": "FeatureCollection",
-    "properties": {},
-    "features": [{
-      "type": "Feature",
-      "geometry": {"type": "Point", "coordinates": [901624.4172679152, 5858812.695644485]},
-      "properties": {
-        "title": ${dumps(locale['title']) | n},
-        "lang": "${locale['lang']}",
-        "documentId": ${image['document_id']},
-        "module": "images"
+  module.value('mapFeatureCollection',
+  % if geometry:
+    {
+      "type": "FeatureCollection",
+      "properties": {},
+      "features": [
+      {
+        "type": "Feature",
+        "geometry": {"type": "Point", "coordinates": [${geometry.x}, ${geometry.y}]},
+        "properties": {
+          "title": ${dumps(locale['title']) | n},
+          "lang": "${locale['lang']}",
+          "documentId": ${image_id},
+          % if image['elevation'] is not None:
+            "elevation": ${image['elevation']},
```

Comment:
There's no elevation for images. At least it's not a relevant info, no need to pass it to the FeatureCollection.